docs/book/getting-started/zenml-pro/organization.md

```diff
@@ -0,0 +1,49 @@
+# Organizations
+
+ZenML Pro arranges various aspects of your work experience around the concept
+of an **Organization**. This is the top-most level structure within the ZenML Cloud environment.
+Generally, an organization contains a group of users and one or more [tenants](../../../../docs/book/getting-started/zenml-pro/tenants.md).
+
+## Create a Tenant in your organization
+
+A tenant is a crucial part of your Organization and holds all of your pipelines, experiments and models, among other things. You need to have a tenant to fully utilize the benefits that ZenML Pro brings. The following is how you can create a tenant yourself:
```

Comment:
Maybe it's worth adding that people can think of a tenant as a single VM / deployment of the ZenML server?